vgen/src/test/java/io/gapi/vgen/testdata/go_library.baseline

```diff
@@ -191,14 +189,14 @@ func (book Book) Path() (string, error) {
 
 // CreateShelf creates a shelf, and returns the new Shelf.
 func (client *Client) CreateShelf(ctx context.Context, req *google_example_library_v1.CreateShelfRequest, opts ...gax.CallOption) (*google_example_library_v1.Shelf, error) {
-	callOpts := append(nonIdempotentRetryCodes(), defaultRetryParams(), client.CallOptions["CreateShelf"]...)
-	callOpts = append(callOpts, opts...)
+	defaultCallOpts := append(nonIdempotentRetryCodes(), defaultRetryParams()...)
+	callOpts := append(client.CallOptions["CreateShelf"], opts...)
```

Comment:
Wow yeah nice catch!

On inspection I think the right course of action is to not export `client.CallOptions` given that it really shouldn't be modifiable outside of initializing a new client struct.

There's still some work to be done on client configuration before it reaches parity with the python implementation... for now I'll make that struct var private.